layer/runtime_layer_data.cc

```diff
@@ -105,14 +121,30 @@ void RuntimeLayerData::LogAndRemoveQueryPools() {
       continue;
     }
 
+    uint64_t invocations[2] = {};
+    result = (query_pool_results_function)(
+        device, info->stat_pool,
+        /*firstQuery=*/0, /*queryCount=*/1,
+        /*dataSize=*/sizeof(invocations), /*pData=*/invocations,
+        /*stride=*/sizeof(invocations[0]),
+        VK_QUERY_RESULT_64_BIT | VK_QUERY_RESULT_WAIT_BIT);
+    if (result != VK_SUCCESS) {
+      discard_result = true;
+    }
+
     if (!discard_result) {
       // FIXME: We should adjust the elapsed units to account for the current
       // GPU frequency. Calling vkGetPhysicalDeviceProperties here causes the
       // driver to crash, however.
-      Log(GetPipelineHash(info->pipeline), timestamp1 - timestamp0);
+      char buffer[256];
+      snprintf(buffer, sizeof(buffer), "%" PRIu64 ",%" PRIu64 ",%" PRIu64,
+               timestamp1 - timestamp0, invocations[0], invocations[1]);
+      buffer[sizeof(buffer) - 1] = 0;
```

Comment:
Maybe zero-initialize the whole buffer, or just stringstream or some formatters from absail?

Reply:
switched to StrCat